packages/components/src/internal/components/productnavigation/ProductLKSDrawer.tsx

```diff
@@ -1,66 +1,81 @@
 import React, { FC, memo, useCallback, useEffect, useMemo, useState } from 'react';
 import { getServerContext } from '@labkey/api';
 
-import { Container, buildURL } from '../../..';
+import classNames from 'classnames';
+
+import { buildURL, incrementClientSideMetricCount } from '../../..';
 
 import { ContainerTabModel } from './models';
-import { LK_DOC_FOLDER_TABS } from './constants';
+import {
+    APPLICATION_NAVIGATION_METRIC,
+    LK_DOC_FOLDER_TABS,
+    TO_LKS_CONTAINER_METRIC,
+    TO_LKS_HOME_METRIC,
+    TO_LKS_TAB_METRIC,
+} from './constants';
 import { ProductClickableItem } from './ProductClickableItem';
 
 interface ProductLKSDrawerProps {
-    /**
-     * List of projects which the current user has permissions to so we can use that to decide which container-item
-     * options to show below.
-     */
-    projects: Container[];
+    showHome: boolean;
+    disableLKSContainerLink?: boolean;
     tabs: ContainerTabModel[];
 }
 
 export const ProductLKSDrawer: FC<ProductLKSDrawerProps> = memo(props => {
-    const { projects, tabs } = props;
-    const { project, container, homeContainer, user } = getServerContext();
-    const showHome = useMemo(() => isProjectAvailable(projects, undefined, 'home'), [projects]);
-    const showProject = useMemo(
-        () =>
-            project !== undefined &&
-            isProjectAvailable(projects, project.id) &&
-            project.name !== homeContainer &&
-            container.path !== '/home',
-        [projects, project, container, homeContainer]
-    );
-    const showContainer = useMemo(() => project !== undefined && project.id !== container.id, [projects, project]);
-
+    const { tabs, disableLKSContainerLink, showHome } = props;
+    const { container, homeContainer, user } = getServerContext();
+    const isHomeContainer = useMemo(() => container.path === '/home', [container]);
```

Comment:
this code matches what was previously there, but since we do have `homeContainer` do you think we should use that in this check?